tornado/ioloop.py

```diff
@@ -1161,19 +1162,28 @@ class PeriodicCallback(object):
     Note that the timeout is given in milliseconds, while most other
     time-related functions in Tornado use seconds.
 
+    If ``jitter`` is specified, each callback time will be randomly selected
+    between ``callback_time`` and ``callback_time * (1 + jitter)``.
```

Comment:
What do you think about making `callback_time` the average time instead of the minimum? That makes it easier to think about the expected rate over time.

Reply:
Makes sense. Done. Now it's callback_time ± jitter/2.